Formula/wires.rb

```diff
@@ -1,8 +1,9 @@
 class Wires < Formula
   desc "WebDriver <-> Marionette proxy"
-  homepage "https://github.com/jgraham/wires"
-  url "https://github.com/jgraham/wires/archive/v0.6.2.tar.gz"
-  sha256 "bbc73c1014dd218b424a82da6e656f638aa672fd0990229eff8c1a005166db07"
+  homepage "https://github.com/mozilla/geckodriver"
+  url "https://github.com/mozilla/geckodriver/archive/v0.8.0.tar.gz"
+  version "0.8.0"
```

Comment:
You can drop this line. The version is automatically inferred from the `url` field.